### PR TITLE
compile at android-studio

### PR DESCRIPTION
**Pull request by talregev**

### CODE REVIEW

**Comment by vrabaud** [comment]
@stonier  , I confirm this works for me too.

File: build.gradle [hunk 1]
```diff
@@ -13,38 +13,44 @@
...
     apply plugin: 'ros-android'
+
+    afterEvaluate { project ->
+        android {
+
+            packagingOptions {
+                /* https://github.com/rosjava/android_core/issues/194 */
+                exclude 'META-INF/LICENSE.txt'
+                exclude 'META-INF/NOTICE.txt'
+                exclude 'META-INF/gradle-plugins/catkin.properties'
```
Comment (stonier) [inline build.gradle:47]: Where are these properties files coming from?

File: gradle.properties [hunk 2]
```diff
@@ -1,3 +0,0 @@
-org.gradle.daemon=true
```
Comment (stonier) [inline gradle.properties:1]: There a reason you wouldn't use the gradle daemon on the command line?

Comment (talregev) [inline gradle.properties:1]: It auto file created with andoird studio. it compile without this.
you can keep it if you like.
what that mean gradle daemon?

File: build.gradle (hunk 1 above)
Comment (talregev) [inline build.gradle:47]: I am not really sure,
It start to appear when you updated bootstrap. 
are you try to compile this project with catkin_make?

File: gradle.properties (hunk 2 above)
Comment (stonier) [inline gradle.properties:1]: The gradle daemon caches oft repeated build instructions. Great for speedups. Android studio already has a daemon by default. Command line doesn't and needs this file to tell it to use a gradle daemon.
- http://jimulabs.com/2014/10/speeding-gradle-builds/

File: build.gradle (hunk 1 above)
Comment (stonier) [inline build.gradle:47]: catkin_make...not seeing a problem yet though. Give me a moment.

File: gradle.properties (hunk 2 above)
Comment (talregev) [inline gradle.properties:1]: ok :)
i didn't notice. so need to add this file for all the projects.

**Comment by stonier** [comment]
Ok...merged this in and then fixed a few things post merging:
- 6201c1ba : deprecated application management as we discussed (its hydro, replaced by common_tools)
- 8486b258 : touched the base build gradle rules to sync with other android stacks
- a300ee69 : most packages were still depending on hydro versions, these had to be updated
- c758adea : fix min sdk version problems, and disabled all but teleop

Indigo needs an update to the map/camera views - @damonkohler changed some code there in hydro. So I've disabled them temporarily from the build in `settings.gradle` till we get a chance to fix that.

**Comment by stonier** [comment]
Thanks @talregev!

:neckbeard: finally everything is at least...roughly building for indigo! :neckbeard:

File: build.gradle (hunk 1 above)
Comment (talregev) [inline build.gradle:47]: @stonier 
this files came from here:
https://github.com/rosjava/rosjava_bootstrap/tree/indigo/gradle_plugins/src/main/resources/META-INF/gradle-plugins

**Pull request merged**